CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ message("   branch : ${GIT_BRANCH}")
 message("     hash : ${GIT_COMMIT_HASH}")
 message("     date : ${BUILD_DATE}")
 
+set(ZITI_HEADER_FILES ${PROJECT_SOURCE_DIR}/includes/nf/errors.h ${PROJECT_SOURCE_DIR}/includes/nf/ziti.h)
```

Comment:
this is no longer used, right?

Reply:
it's referenced in library/CMakeLists.txt but not in the example programs

Reply:
it should be moved to the one in library cmake then

Reply:
moved